### PR TITLE
feat: shrink and enforcer convert all hostiles

### PR DESCRIPTION
**Pull request by Golbolco**
Separated out the CureAllMadness card into two cards: CureAllMadness and StopConversions

StopConversions now converts all hostiles to Villager/Traitor depending on the role

### CODE REVIEW

File: Games/types/Mafia/roles/cards/StopConversion.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+module.exports = class CureAllMadness extends Card {
+  constructor(role) {
+    super(role);
+
+    this.meetings = {
+      "Prevent Conversion": {
+        states: ["Night"],
+        flags: ["voting"],
+        action: {
+          priority: PRIORITY_EFFECT_GIVER_DEFAULT + 1,
```
Comment (pikulet) [inline Games/types/Mafia/roles/cards/StopConversion.js:13]: it should be convert priority minus a certain value

File: Games/types/Mafia/roles/cards/CureAllMadness.js [hunk 2]
```diff
@@ -5,32 +5,16 @@ module.exports = class CureAllMadness extends Card {
...
     super(role);
 
-    this.meetings = {
-      "Cure Madness": {
-        states: ["Night"],
-        flags: ["voting"],
-        action: {
+    this.actions = [
+      {
           priority: PRIORITY_EFFECT_GIVER_DEFAULT + 1,
```
Comment (pikulet) [inline Games/types/Mafia/roles/cards/CureAllMadness.js:10]: effect taker default?

Comment (Golbolco) [reply Games/types/Mafia/roles/cards/CureAllMadness.js:10]: there is no Priority_effect_taker_default, this card always used effect_giver. Would you want me to create a new priority? can do

**Comment by pikulet** [comment]
@Golbolco you cannot separate the two cards if they rely on the same target, unless cure all madness is another meeting

**Pull request merged**